### DISCUSSION TITLE
createSpeech

### DISCUSSION THREAD

**Discussion by knuppe** [allanbunch/node-red-openai-api#12] [Q&A] [answered]
How does the `createSpeech` component work? I couldn't make it save an MP3 file; from what I've noticed, the output of the node is a string. When I try to save it as an MP3, the file becomes corrupted.

**Comment by allanbunch** [comment]
I see the issue. I'll drop a note here once it's resolved. Stay tuned.

**Comment by allanbunch** [comment, marked as answer]
@knuppe - This is fixed in the latest published version: 0.4.0. Thanks for the nudge.
